src/prefect/utilities/executors.py

```diff
@@ -174,11 +174,42 @@ def multiprocessing_safe_run_and_retrieve(
         with prefect.context(context):
             logger.debug(f"{name}: Executing...")
             return_val = fn(*args, **kwargs)
+            logger.debug(f"{name}: Execution successful.")
     except Exception as exc:
         return_val = exc
+        logger.error(
+            f"{name}: Encountered a {type(exc).__name__}, "
+            f"returning details as a result..."
+        )
```

Comment:
Can't get your comment to come up here so replying in a new one -- adding `exc_info` here is unnecessary because this exception will be pickled and returned to be re-raised in the parent function.

Reply:
Hm I get your reasoning but that seems like a brittle way to lose this info if that ever changes.  I guess any logs here are better than the previous behavior so I won't push the point

Reply:
👍 I think the next time someone encounters issues here will tell us if this is sufficient or if we need to do more.  When I first started working on this, there were no logs at all and we've been trending gradually verbose.

If the behavior changes above this (such that the exception isn't raised) we'd have big problems because then any errors in the `task.run()` function would also be silenced, so I'm not concerned about brittleness there.